e2e-tests/service-per-pod/run

```diff
@@ -20,7 +20,7 @@ check_cr_config() {
 	compare_kubectl service/$cluster-0
 
 	local URI="$(get_service_ip $cluster-0),$(get_service_ip $cluster-1),$(get_service_ip $cluster-2)"
-	sleep 20
+	sleep 30
```

Comment:
please add comment what are we waiting for?

Reply:
done